<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>651</width>
     <height>780</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="username">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>340</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>굴림</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>사용자 이름</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_enter">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>420</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QListWidget" name="chatroom">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>510</width>
       <height>591</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>채팅 목록</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_exit">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>나가기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_makechat">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>80</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>채팅방 만들기</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newchatname">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>86</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btn_send">
     <property name="geometry">
      <rect>
       <x>499</x>
       <y>687</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>전송</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sendmessage">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>686</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
    <widget class="QListWidget" name="receivemessage">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>159</y>
       <width>521</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="chatmember">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>접속인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="chat_title">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
